app/models/post.rb

```diff
@@ -49,6 +49,8 @@ class Post < ApplicationRecord
   belongs_to :spoiled_unit, polymorphic: true
   has_many :post_likes, dependent: :destroy
   has_many :comments, dependent: :destroy
+  # TODO: what happens if the original is destroyed?
```

Comment:
oh reblogs are not like an embedded post, it is just a reference to the original, got it.

Also, what about editing the original post? Are we just going to go with once it is reblogged all editing is disabled?